weather_dl_v2/cli/app/subcommands/license.py

```diff
@@ -26,6 +26,11 @@ class LicenseValidator(Validator):
     pass
 
 
+license_key_order = [
+    'secret_id', 'k8s_deployment_id', 'status', 'client_name', 'license_id', 'number_of_requests'
```

Comment:
nit: license_id, client_name, status, number_of_requests, secret_id, k8s_deployment_id.

Reply:
updated in [7887b24](https://github.com/google/weather-tools/pull/424/commits/7887b243c8daf71792e56390c2f561f307bf24f0)